src/js/edu-benefits/utils/veteran.js

```diff
@@ -178,3 +184,130 @@ export function createVeteran() {
     }
   };
 }
+
+function convertFieldToValue(field) {
```

Comment:
I updated this to match the hca approach.

I really dislike Javascript's case statements, but it is simpler.